aggregators/bulbaswap/index.ts

```diff
@@ -0,0 +1,112 @@
+import { ChainBlocks, FetchOptions, SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import { httpPost } from "../../utils/fetchURL";
+
+const v2Fees = 0.0035;
+
+// V2 endpoints
+const v2Endpoints = "https://api.bulbaswap.io/v1/subgraph-apis/v2";
+
+// V3 endpoints
+const v3Endpoints = "https://api.bulbaswap.io/v1/subgraph-apis/v3";
+
+// GraphQL queries
+const factoryQuery = `{
+  uniswapFactory(id: "0x8D2A8b8F7d200d75Bf5F9E84e01F9272f90EFB8b") {
+    totalLiquidityUSD
+    totalVolumeUSD
+    txCount
+  }
+  uniswapDayDatas(first: 1, orderBy:date, orderDirection:desc) {
+    dailyVolumeUSD
+    date
+  }
+}`;
+
+const v3FactoryQuery = `{
+  factory(id: "0xFf8578C2949148A6F19b7958aE86CAAb2779CDDD") {
+    totalValueLockedUSD
+    totalVolumeUSD
+    totalFeesUSD
+    txCount
+  }
+  uniswapDayDatas(first: 1, orderBy: date, orderDirection: desc) {
```

Comment:
could you use query by Id, because if data not up to date will collect incorrect data